samples/subsys/zbus/msg_subscriber/README.rst

```diff
@@ -0,0 +1,78 @@
+.. zephyr:code-sample:: zbus-msg-subscriber
+   :name: zbus Message subscriber
+   :relevant-api: zbus_apis
+
+   Use zbus channels with message subscribers.
```

Comment:
Nit: I think the wording should be swapped (the sample is supposed to be about subscribers, not channels), and augmented to better reflect what I understand is the main feature introduced by subscribers. Please ignore if I got it wrong :)

```suggestion
   Use zbus message subscribers to listen to messages published to channels.
```